src/data/calculate.ts

```diff
@@ -54,6 +57,8 @@ export const defaultValues: CalculatorData = {
   theirMask: '',
   yourMask: '',
   voice: '',
+
+  scenarioName: '',
```

Comment:
unnecessary, since it will default to undefined.

Reply:
default to undefined?

Reply:
We use this to reset the form when someone presses the reset button. So I think it is necessary for that case. Am I missing something?